src/components/Button/index.js

```diff
@@ -9,6 +9,7 @@ const Button = ({
   children,
   className,
   onClick,
+  primary,
```

Comment:
why did you add this prop again?

Reply:
ah, good catch.  to get the errors to stop.  it was meant to be temporary and never merged in.  i will fix this now